src/objects/Funding_Request__c.object

```diff
@@ -197,6 +197,14 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
```

Comment:
Just confirming that we aren't (can't?) deploying the setting to populate existing records.